### PR TITLE
Fix console errors

### PR DESCRIPTION
**Pull request by mdubus**
# Pull Request

## What does this PR do?
- Fix console errors which happened inside the Lottie component

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: src/components/Lottie.js [hunk 1]
```diff
@@ -30,10 +30,9 @@ const LottieAnimation = ({
       options={lottieOptions}
       height="100%"
       width="100%"
-      ariaRole={null}
       ariaLabel={ariaLabel}
       tabIndex={-1}
-      className={hideInPercy && 'hide-in-percy'}
+      className={hideInPercy ? 'hide-in-percy' : undefined}
```
Comment (bidoubiwa) [inline src/components/Lottie.js:35]: Is your className value `undefined` ? How does it appear in the final dom element?

Comment (mdubus) [reply src/components/Lottie.js:35]: I had the following console error: 
`Warning: Received `false` for a non-boolean attribute `className`.

If you want to write it to the DOM, pass a string instead: className="false" or className={value.toString()}.

If you used to conditionally omit it with className={condition && value}, pass className={condition ? value : undefined} instead.`

So I just applied their suggestion in the last sentence :)

**Review by bidoubiwa — APPROVED** [review]
🔥

**Comment by mdubus** [comment]
bors merge

**Pull request merged**